blocks/init/src/Blocks/components/modal/modal-style.scss

```diff
@@ -0,0 +1,88 @@
+.modal {
+	$this: &;
+
+	--modal-scoped-icon-size: 1.5rem;
+
+	display: none;
+
+	&.is-open {
+		display: block;
+	}
+
+	&__overlay {
+		position: fixed;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		z-index: 999;
+		background-color: rgba(0, 0, 0, 0.5);
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		will-change: transform;
+	}
+
+	&__dialog {
+		min-width: 360px;
+		max-width: calc(100% - 1.25rem);
+		max-height: 90%;
+		overflow: auto;
+		position: relative;
+		padding: 1.25rem;
+		will-change: transform;
+		background-color: var(--global-colors-white);
+	}
+
+	&__close-button {
+		@extend %button-reset;
+		height: var(--modal-scoped-icon-size);
+		width: var(--modal-scoped-icon-size);
+
+		svg {
+			height: var(--modal-scoped-icon-size);
+			width: var(--modal-scoped-icon-size);
+			pointer-events: none;
+		}
+	}
+
+	@keyframes modalFadeIn {
+		from { opacity: 0; }
+		  to { opacity: 1; }
+	}
+
+	@keyframes modalFadeOut {
+		from { opacity: 1; }
+		  to { opacity: 0; }
+	}
+
+	@keyframes modalSlideIn {
+	  from { transform: translateY(15%); }
+		to { transform: translateY(0); }
+	}
+
+	@keyframes modalSlideOut {
+		from { transform: translateY(0); }
+		  to { transform: translateY(-10%); }
+	}
+
+	&[aria-hidden='false'] {
+		#{$this}__overlay {
+			animation: modalFadeIn 0.3s cubic-bezier(0, 0, 0.2, 1);
+		}
+	
+		#{$this}__dialog {
+			animation: modalSlideIn 0.3s cubic-bezier(0, 0, 0.2, 1);
+		}
+	}
+
+	&[aria-hidden='true'] {
+		#{$this}__overlay {
+			animation: modalFadeOut 0.3s cubic-bezier(0, 0, 0.2, 1);
+		}
+	
+		#{$this}__dialog {
+			animation: modalSlideOut 0.3s cubic-bezier(0, 0, 0.2, 1);
+		}
+	}
```

Comment:
You should probably be able to use CSS transitions for this (depending on what you do with the `display` property)

Reply:
I see what you mean and that’s true
It can’t be animated, but the library can handle it that way
But anyway, since library could be replaced, I’ll change that